src/sidebar/CustomModelBox.tsx

```diff
@@ -76,9 +77,16 @@ export default function CustomModelBox({ enabled, encodedValues }: CustomModelBo
         // we start with empty categories. they will be set later using info
         const instance = create({}, (element: Node) => divElement.current?.appendChild(element))
         setEditor(instance)
-        setCustomModelExample(instance, examples['empty'])
-        // todo: minor glitch: if the initial model is invalid we see an 'Invalid custom model' error notification, even
-        //       though the custom model box is closed initially
+
+        if (initialCustomModelStr != null) {
+            // prettify the custom model if it can be parsed or leave it as is otherwise
+            try {
+                initialCustomModelStr = customModel2prettyString(JSON.parse(initialCustomModelStr))
+            } catch (e) {}
```

Comment:
In our initial implementation (some commit in #183) the app crashed when the string was no valid json. Now we catch this error here and simply paste it as given to the custom model box.